jaxrs/src/main/java/com/cerner/beadledom/jaxrs/exceptionmappers/FailureExceptionMapper.java

```diff
@@ -0,0 +1,70 @@
+package com.cerner.beadledom.jaxrs.exceptionmappers;
+
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+
+import com.cerner.beadledom.jaxrs.models.JsonError;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import org.jboss.resteasy.spi.Failure;
```

Comment:
Since this class depends on resteasy classes it belongs in the resteasy module. This will resolve @shiftyeyes comment [1] as well.

[1] https://github.com/cerner/beadledom/pull/45/files#r136570400

Reply:
I moved the `FailureExceptionMapper` to the `beadledom-resteasy` module: https://github.com/cerner/beadledom/pull/45/commits/ff9039283a54bd60c160857d65055c3ba9d1b35f